sdk/python/tests/connectors/test_docker.py

```diff
@@ -45,3 +70,90 @@ async def connect_once():
         next(files)
     # assert the garbage was cleaned up
     assert not garbage_path.exists()
+
+
+def test_docker_cli_is_not_installed(
```

Comment:
These tests are marked with `anyio` from the module (i.e., _async_). Sync tests should remove the mark or go into a separate `test_docker_sync.py` file.

Reply:
I opted to keep all the tests in the same file, it seemed less confusing with the `sync`/`async` apis.

Reply:
What I mean is that there's module wide [marks](https://docs.pytest.org/en/7.2.x/how-to/mark.html) in this file for `anyio` and `slow`:

```python
pytestmark = [
    pytest.mark.anyio,
    pytest.mark.slow,
]
```

Tests should be marked appropriately, so either put them in each test or split in different files.

Reply:
Sorry, I should have clarified (or waited till I pushed the commit), I kept all the tests in the same file & marked each test appropriately.